### ISSUE TITLE
Support Jdbc batch insert with setAutoCommit(false) as it used by Apache Spark

### ISSUE THREAD

**Issue by igor-suhorukov** [questdb/questdb#145]
**Is your feature request related to a problem? Please describe.**
Please support transaction or just ignore [BEGIN](https://www.postgresql.org/docs/12/sql-begin.html) SQL keyword when Jdbc driver setAutoCommit(false) and [COMMIT](https://www.postgresql.org/docs/12/sql-commit.html) & [ROLLBACK](https://www.postgresql.org/docs/12/sql-rollback.html) should be support by the same implementation approach

**Describe the solution you'd like**
Jdbc batch insert is common vendor neutral feature in JVM ecosystem for data import  that supported by plenty number of java frameworks. Also Apache Spark use it in jdbc.write into PG driver.

**Describe alternatives you've considered**
Native support in QuestDB of parquet, avro, orc format by reading data from HDFS,S3, Azure Storage with complex path/naming patterns.

**Additional context**
[PostgreSqlBatchInsertTest ](https://github.com/igor-suhorukov/questdb-issue/blob/master/src/test/java/io/questdb/sql/PostgreSqlBatchInsertTest.java) should pass when this feature will be implemented
```
git clone https://github.com/igor-suhorukov/questdb-issue.git
cd questdb-issue
mvn test -Dtest=io.questdb.sql.PostgreSqlBatchInsertTest
```

**Comment by igor-suhorukov** [comment]
To suppress transaction command in spark jdbc write command it is possible to set isolationLevel to NONE:

parquet.write().mode(SaveMode.Overwrite).option("truncate", true).format("jdbc")
                            .option("url", "jdbc:postgresql://localhost:8812/")
                            .option("dbtable", "t1")
                            .option("isolationLevel", "NONE")
...
                            .save();

But import still not work

**Comment by igor-suhorukov** [comment]
Code to reproduce io.questdb.sql.PostgreSqlBatchInsertTest https://github.com/igor-suhorukov/questdb-issue

batchInsert.executeLargeBatch() works fine
batchInsert.executeBatch() is hang

**Comment by igor-suhorukov** [comment]
https://github.com/igor-suhorukov/spark3

by default runs on standard dependencies, but with patched pg jdbc driver(postgresql-42.3.0-SNAPSHOT.jar) and spark(spark-sql_2.12-3.1.0-SNAPSHOT.jar) it runs with incomplete data import:

![Screenshot from 2020-04-18 23-25-49](https://user-images.githubusercontent.com/10332206/79671029-cdd93900-81cf-11ea-806e-aed77cf775d8.png)

![Screenshot from 2020-04-18 23-41-22](https://user-images.githubusercontent.com/10332206/79671028-cade4880-81cf-11ea-8edb-cf9e96c9b079.png)
